onnxruntime/contrib_ops/cpu/transformers/beam_search_device_helper.cc

```diff
@@ -211,6 +212,21 @@ void InitBeamState(transformers::IBeamSearchState<T>* beam_state,
   }
 }
 
+template <typename T>
+void InitGreedyState(transformers::IGreedySearchState<T>* greedy_state,
```

Comment:
shall we rename beam_search_device_helper.cc or move greedy search functions to another file like greedy_search_device_helper.cc?

Reply:
how about generation_device_helper?